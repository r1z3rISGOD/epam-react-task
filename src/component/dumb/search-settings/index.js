import {SearchSettings} from "./search-settings"
export {SearchSettings}
