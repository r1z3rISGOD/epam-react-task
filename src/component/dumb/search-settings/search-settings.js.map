{"version":3,"file":"search-settings.js","sourceRoot":"","sources":["search-settings.tsx"],"names":[],"mappings":";;;AAAA,6BAA8B;AAC9B,kDAA+C;AAC/C,kDAA+C;AAC/C,kCAA+B;AAUxB,IAAM,cAAc,GAAmC,UAAC,EAA8D;QAA5D,OAAO,aAAA,EAAE,OAAO,aAAA,EAAE,UAAU,gBAAA,EAAE,WAAW,iBAAA,EAAE,gBAAgB,sBAAA;IAC1H,OAAO,CACD,6BAAK,SAAS,EAAC,iBAAiB;QAC5B,oBAAC,4BAAY,IACT,UAAU,EAAE,UAAU,EACtB,OAAO,EAAE,OAAO,GAClB;QACF,oBAAC,4BAAY,IACT,WAAW,EAAE,WAAW,EACxB,OAAO,EAAE,OAAO,EAChB,OAAO,EAAE,OAAO,EAChB,gBAAgB,EAAE,gBAAgB,GACpC,CACA,CACX,CAAA;AACH,CAAC,CAAA;AAfY,QAAA,cAAc,kBAe1B"}