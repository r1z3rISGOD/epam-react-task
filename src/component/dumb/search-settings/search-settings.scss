.search-settings {
  display: flex;
  flex-direction: row;
  justify-content: space-between;
  align-items: center;
  margin-top: 15px;
  padding-bottom: 50px;
}
