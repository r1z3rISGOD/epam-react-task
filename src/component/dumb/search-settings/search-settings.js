"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.SearchSettings = void 0;
var React = require("react");
var search_filter_1 = require("../search-filter");
var search_button_1 = require("../search-button");
require("./search-settings.scss");
var SearchSettings = function (_a) {
    var request = _a.request, setting = _a.setting, setSetting = _a.setSetting, resultsSort = _a.resultsSort;
    return (React.createElement("div", { className: "search-settings" },
        React.createElement(search_filter_1.SearchFilter, { setSetting: setSetting, setting: setting }),
        React.createElement(search_button_1.SearchButton, { resultsSort: resultsSort, request: request, setting: setting })));
};
exports.SearchSettings = SearchSettings;
//# sourceMappingURL=search-settings.js.map