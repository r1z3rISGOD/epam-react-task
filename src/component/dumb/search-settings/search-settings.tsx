import * as React from 'react'
import { SearchFilter } from '../search-filter'
import { SearchButton } from '../search-button'
import './search-settings.scss'

interface SearchSettingsProps {
    request: string
    setting: string
    setSetting: any
    resultsSort: string
    fetchingAllFilms: any
}

export const SearchSettings : React.FC<SearchSettingsProps> = ({ request, setting, setSetting, resultsSort, fetchingAllFilms}) => {
  return (
        <div className="search-settings">
            <SearchFilter
                setSetting={setSetting}
                setting={setting}
            />
            <SearchButton
                resultsSort={resultsSort}
                request={request}
                setting={setting}
                fetchingAllFilms={fetchingAllFilms}
            />
        </div>
  )
}
