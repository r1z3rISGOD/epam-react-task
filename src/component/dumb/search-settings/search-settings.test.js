import { SearchSettings } from './search-settings.tsx'
import React from 'react'

it('search settings', () => {
  const comp = shallow(<SearchSettings/>)
  expect(comp).toMatchSnapshot()
})
