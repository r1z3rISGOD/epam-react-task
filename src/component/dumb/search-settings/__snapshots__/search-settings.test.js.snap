// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`search settings 1`] = `
<div
  className="search-settings"
>
  <SearchFilter />
  <SearchButton />
</div>
`;
